audio-api/src/main/scala/no/ndla/audioapi/AudioApiProperties.scala

```diff
@@ -36,6 +37,13 @@ class AudioApiProperties extends BaseProps with DatabaseProps with StrictLogging
   val StorageName: String           = propOrElse("AUDIO_FILE_S3_BUCKET", s"$Environment.audio.ndla")
   val StorageRegion: Option[String] = propOrNone("AUDIO_FILE_S3_BUCKET_REGION")
 
+  val TranscribeStorageName: String = propOrElse("TRANSCRIBE_FILE_S3_BUCKET_NAME", s"$Environment.transcribe.ndla")
+  val TranscribeStorageRegion: Option[String] = propOrNone("TRANSCRIBE_FILE_S3_BUCKET_REGION")
+
+  val BrightcoveClientId: String     = prop("BRIGHTCOVE_API_CLIENT_ID")
+  val BrightcoveClientSecret: String = prop("BRIGHTCOVE_API_CLIENT_SECRET")
+  val BrightcoveAccountId: String    = prop("NDLA_BRIGHTCOVE_ACCOUNT_ID")
```

Comment:
Denne finnes allerede med dette navnet i secrets
```suggestion
  val BrightcoveAccountId: String    = prop("BRIGHTCOVE_ACCOUNT_ID")
```